### PR TITLE
Ek hugo translate

### PR DESCRIPTION
**Pull request by ekerrison**
Translated workshop to Hugo

### CODE REVIEW

File: content/0-setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
++++
+title = "Setup"
+weight = 1
+chapter = true
++++
+
+# Setup
+
+{{%attachments style="grey" /%}}
```
Comment (Dadams2) [inline content/0-setup/_index.md:7]: this line causes an error in my hugo build please remove 
```Error: Error building site: "intro-unix-shell/content/0-setup/_index.md:9:1": failed to extract shortcode: template for shortcode "attachments" not found```

**Comment by Dadams2** [comment]
also you probably want to remove the `.bundle` and `.vendor` files

File: content/0-setup/_index.md (hunk 1 above)
Comment (ekerrison) [reply content/0-setup/_index.md:7]: hm that's odd, the attachments shortcode is part of the learn theme so it shouldn't cause an error. I'll look into this

Comment (Dadams2) [reply content/0-setup/_index.md:7]: im probably doing something dumb then. Ill let you know if i fix it

Comment (Dadams2) [reply content/0-setup/_index.md:7]: learn wasn't installed as it wasn't added as a submodule. My bad. I might want to fix this in the template though at some point

**Pull request merged**